collector/accelerators_test.go

```diff
@@ -68,3 +74,15 @@ func TestAccelerator(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func Test_prepareVendorModelData(t *testing.T) {
+	_, err := prepareVendorModelData("accelerators_test_data_multiple_vendors.yaml")
+	if err == nil {
+		t.Fatal(err)
+	}
+
+	_, err = prepareVendorModelData("testdata/accelerators_test_data_multiple_device_ids.yaml")
```

Comment:
```suggestion
	_, err = prepareVendorModelData("testdata/accelerators_test_data_duplicated_device_ids.bad.yaml")
```

To make it clearer the config loading should fail.

Reply:
fixed